### ISSUE TITLE
Add module to npm registry

### ISSUE THREAD

**Issue by rroblik** [jawj/OverlappingMarkerSpiderfier#125]
Hi Jawj !

Thanks for all hard work done, and more recently with the 1.0 release.

I used a fork of your dev in a webpack project (https://www.npmjs.com/package/overlapping-marker-spiderfier) because I didn't found your on npm registry.

As now your library can be loaded asynchronously form Google Maps, do you think you can add it to npm registry ?

that can be cool to easily integrate it !

Regards

**Comment by jawj** [comment]
You’re welcome!

I’m late to the party of browser-side dependency management. To clarify, is there any value publishing an npm package if I don’t also wrap stuff up to enable browserify (for example) to do its thing?

**Comment by rroblik** [comment]
I think it could be done in multiple steps :)
Regards

**Comment by matpb** [comment]
+1 for the recent love given to OMS, thank you Jawj!

I was recently working on an NPM module for OMS and here is what I came up with, if that can give you some ideas: https://pastebin.com/7bYgf9aa

I added an argument so that it can be loaded async and feed the window.google object to the new OMS instance.

I also played with a new function "spiderfyAll" to always spiderfy all visible markers (required for a project) but it's quite hacky...

**Comment by danielkoch** [comment]
Any updates here? I found this one: https://www.npmjs.com/package/overlapping-marker-spiderfier But I'm not quite sure if this is maintained further more.

**Comment by SerkanSipahi** [comment]
any updates for npm support?

**Comment by jamendub** [comment]
I would love to see this happen to !
Thanks anyway for the library man !

**Comment by strarsis** [comment]
@jawj: Yes! I am not able to find a built JavaScript library from this, installing from npm would be best practice.
